RocketChat/Rocket.Chat
# feat: create contact endpoint

## Proposed changes (including videos or screenshots)

Implement the entity contact (collection `rocketchat_livechat_contact`) to the omnichannel and an endpoint to be able to create a contact manually.

[POST] `api/v1/omnichannel/contacts`
request example:
```json
{
	"name": "Contact 1",
	"emails": [
		"contact1@example.com"
	],
	"phones": [
		"5511999999999"
	],
}
```
response example:
```json
{
	"contactId": "668830239ac5cb4b15229d6d",
	"success": true
}
```

## Issue(s)


## Steps to test or reproduce

1. create a contact using the new endpoint mentioned above;
2. check on the collection;

## Further comments

[SCI-33](https://rocketchat.atlassian.net/browse/SCI-33)


[SCI-33]: https://rocketchat.atlassian.net/browse/SCI-33?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
